float64[6] joint_angles
float64 vel
float64 acc
bool wait_flag